internal/pkg/model/filter.go

```diff
@@ -28,6 +29,10 @@ type ObjectsFilter struct {
 	ignoredComponents ComponentIDs
 }
 
+var alwaysIgnoredComponents = map[string]bool{
```

Comment:
Pokud to dám do `ignoredComponents` tak to bude vidět v manifestu. Stačí to takhle?